cypress/e2e/buy-shirt.cy.ts

```diff
@@ -1,21 +1,37 @@
+import {MenuContentPage, ProductListPage, ShoppingCartPage, LoginPage} from "../page/index";
+import {AddressStepPage, ShippingStepPage, PaymentStepPage} from "../page/index";
+
+const menuContentPage = new MenuContentPage();
+const productListPage = new ProductListPage();
+const shoppingCartPage = new ShoppingCartPage();
+const loginProcess = new LoginPage();
+const addressStep = new AddressStepPage();
+const shippingStep = new ShippingStepPage();
+const paymentStep = new PaymentStepPage();
+
+
 describe("Buy a t-shirt", () => {
+  const email = "aperdomobo@gmail.com";
+  const passwd = "WorkshopProtractor";
+
   it("then the t-shirt should be bought", () => {
-    cy.visit("http://automationpractice.com/");
-    cy.get("#block_top_menu > ul > li:nth-child(3) > a").click();
-    cy.get("#center_column a.button.ajax_add_to_cart_button.btn.btn-default").click();
-    cy.get("[style*='display: block;'] .button-container > a").click();
-    cy.get(".cart_navigation span").click();
-
-    cy.get("#email").type("aperdomobo@gmail.com");
-    cy.get("#passwd").type("WorkshopProtractor");
-
-    cy.get("#SubmitLogin > span").contains("Sign in").click();
-    cy.get(".cart_navigation > .button > span").contains("Proceed to checkout").click();
-    cy.get("#cgv").click();
-    cy.get(".cart_navigation > .button > span").contains("Proceed to checkout").click();
-    cy.get(".bankwire").contains("Pay by bank wire").click();
-    cy.get("#cart_navigation > button > span").contains("I confirm my order").click();
-
-    cy.get("#center_column > div > p > strong").should("have.text", "Your order on My Store is complete.");
+    menuContentPage.visitMenuContentPage();
+    menuContentPage.goToTShirtMenu();
+
+    productListPage.addToCart();
+    productListPage.proceedToCheckOutPopUpWindow();
+
+    shoppingCartPage.proceedToCheckOut();
+
+    loginProcess.login(email, passwd);
+
+    addressStep.proceedToCheckOut();
+
+    shippingStep.checkTS();
+    shippingStep.proceedToCheckOut();
+
+    paymentStep.selectBankWirePayment();
+    paymentStep.confirmOrder();
+    paymentStep.assert();
```

Comment:
make this method to receive the message to assert as parameter. Also you can rename the method as `assertConfirmationMessage`.

Other option, can be, to rename the method as getConfirmationMessage, and perform the assertion in this test file.